src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -2754,6 +2754,9 @@ components:
         resourceGroupName:
           type: string
           description: an optional resource group name for Azure
+        applicationDeploymentName:
+          type: string
+          description: an optional name for an application deployment for Azure
```

Comment:
Does Azure have any requirements around naming, like max length? It could be useful to validate that here

Reply:
It does but this is pointing to an application that already exists, so I'm not sure it would buy us much